CHANGELOG.md

```diff
@@ -1,172 +1,180 @@
 ### Master
 
-#### Emission
-
-- [Messaging] Adds an ArtworkPreview to each Conversation - sarah
-- [Messaging] Adds inbox zero state - maxim
-- [Messaging] Relay support in Conversation container - sarah
-- [Messaging] Adds a FlatList for Messages in Conversations - sarah
-- [dev] Applies prettier, and adds it to the webhooks + extension settings - orta
-- [dev] Adds stories for new messaging components - maxim
-- [Messaging] Adds Conversation, Composer, and Message components - sarah
-- [dev] Upgrades Sentry to 3.0, note Eigen integrator, this may require changes to Eigen - orta
-- [Messaging] Adds Inbox view with real data - luc
-- [dev] Upgrades React Native to v0.45 - sarah
-- [dev] Allows toggling the back button by pressing space - orta
-- Users/Devs can run any PR from inside beta/sim - orta
-- Users now can access stories from inside the app - orta
-
-#### Consignments
-
-- [dev] Adds component for attaching buttons to the keyboard  - orta
-- [dev] Adds component for artist search  - orta
-- [dev] Adds component for consignments todo  - orta
-- [dev] Adds a root component for the Consignments flow  - orta
-- [dev] Adds storybooks for consignments  - orta
-- [dev] Adds some form elements for consignments - orta
-- Added the Welcome/Overview screen for consignments - orta
+-   [dev] Moves native view controllers to use iOS7+ status bar api - ash
+
+###### Emission
+
+-   Users/Devs can run any PR from inside beta/sim - orta
+-   Users now can access stories from inside the app - orta
+-   [dev] Applies prettier, and adds it to the webhooks + extension settings - orta
+-   [dev] Adds stories for new messaging components - maxim
+-   [dev] Upgrades Sentry to 3.0, note Eigen integrator, this may require changes to Eigen - orta
+-   [dev] Upgrades React Native to v0.45 - sarah
+-   [dev] Allows toggling the back button by pressing space - orta
+
+###### Messaging
+
+-   Adds an `ArtworkPreview` to each Conversation - sarah
+-   Adds Inbox zero state - maxim
+-   Relay support in Conversation container - sarah
+-   Adds a `FlatList` for Messages in Conversations - sarah
+-   Adds Conversation, Composer, and Message components - sarah
+-   Adds Inbox view with real data - luc
+
+###### Consignments
+
+-   Adds a root component for the Consignments flow  - orta
+-   Added the Welcome/Overview screen for consignments - orta
+-   Adds component for attaching buttons to the keyboard  - orta
+-   Adds component for artist search  - orta
+-   Adds component for consignments todo  - orta
+-   Adds storybooks for consignments  - orta
+-   Adds some form elements for consignments - orta
 
 ### 1.3.8 & 1.3.9
 
-- [dev] Moves native view controllers to use iOS7+ status bar api - ash
+-   [dev] Moves native view controllers to use iOS7+ status bar api - ash
 
 ### 1.3.6
 
-- Fixes missing compiled bundle - alloy
+-   Fixes missing compiled bundle - alloy
```

Comment:
Good question in general, seems to be recommend as the default in remark: https://github.com/wooorm/remark